### DISCUSSION TITLE
How to use @prisma/nextjs-monorepo-workaround-plugin with Turbo Pack ( next --turbo )

### DISCUSSION THREAD

**Discussion by davecarlson** [prisma/prisma#23853] [Q&A]
### Question

currently, to use the plugin we do : 
```
webpack: (config, { isServer }) => {
    if (isServer) {
      config.plugins = [...config.plugins, prismaPlugin];
    }

    return config;
  }
```

however, this throws issues if you are using turbopack (next --turbo): 

```
⚠ Webpack is configured while Turbopack is not, which may cause problems.
 ⚠ See instructions if you need to configure Turbopack:
https://nextjs.org/docs/app/api-reference/next-config-js/turbo
```

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by ludralph** [comment]
Hi @davecarlson 👋 

Thank you for raising this question.

The `@prisma/nextjs-monorepo-workaround-plugin` was a solution for Next.js monorepo setups. It might not be the best approach with Turbo Pack. The reason is Turbo Pack has built-in support for Prisma within a monorepo structure. This eliminates the need for an additional workaround plugin. Did you take a look at this section of the [Turbo Repo](https://turbo.build/repo/docs/handbook/tools/prisma) documentation on how to setup Prisma with Turbo repo?

**Reply by davecarlson** [reply]
We are already on a turborepo ( have been over a year). ( and had used that guide).

Please note, its absolutely fine on normal build, just not the new --turbo build option ( turbo pack is separate to turbo repo )

**Reply by ludralph** [reply]
Hi @davecarlson 

Are you still experiencing issues with the `@prisma/nextjs-monorepo-workaround-plugin` with Turbo pack?

**Reply by davecarlson** [reply]
Yes, same issues !

**Reply by ludralph** [reply]
Thanks for confirming. I'll share this internally with the team. I'll provide you with any update on this issue.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!;
